data/mods/ssb/statuses.js

```diff
@@ -1203,8 +1203,15 @@ let BattleStatuses = {
 	},
 	pre: {
 		noCopy: true,
-		onStart() {
+		onStart(source) {
 			this.add(`c|~pre|let's go, in and out, 20 minute adventure`);
+			// Easter Egg
+			let activeMon = toID(source.side.foe.active[0].illusion ? source.side.foe.active[0].illusion.name : source.side.foe.active[0].name);
+			if (activeMon === 'anubis') {
+				this.add(`c|&Anubis|ohey it's pre`);
+				this.add(`c|&Anubis|!showimage https://pokemonshowdown.com/images/ssbkitten.jpg,300,400`);
```

Comment:
does this actually get interpeted? just drop the 300,400 - its just noise IMO.

Reply:
No, Anubis and I tested it out in a battle earlier and found out defaults to a smaller size. However if you want it removed fine by me

Reply:
I understand it defaults to smaller, but because the command isnt actually being interpreted (or else why would you add `|raw|`), it doesnt seem necessarily to display (no regs can even use the command to verify we're bending the rules)